connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookConnector.java

```diff
@@ -0,0 +1,76 @@
+package io.camunda.connector.runtime.inbound.webhook;
+
+import io.camunda.connector.api.annotation.InboundConnector;
+import io.camunda.connector.api.inbound.InboundConnectorContext;
+import io.camunda.connector.api.inbound.InboundConnectorExecutable;
+import io.camunda.connector.impl.inbound.InboundConnectorProperties;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+@InboundConnector(name = "WEBHOOK", type = WebhookConnector.TYPE_WEBHOOK)
```

Comment:
Earlier the type was just `webhook`, I think it makes sense to unify this with the outbound connectors. This is just a reminder for me that this must be added to the upgrade guide.